addons/crm/models/crm_lead.py

```diff
@@ -728,10 +728,12 @@ def _read_group_stage_ids(self, stages, domain, order):
         stage_ids = stages._search(search_domain, order=order, access_rights_uid=SUPERUSER_ID)
         return stages.browse(stage_ids)
 
-    def _stage_find(self, team_id=False, domain=None, order='sequence'):
+    def _stage_find(self, team_id=False, domain=None, order='sequence', limit=1):
         """ Determine the stage of the current lead with its teams, the given domain and the given team_id
             :param team_id
             :param domain : base search domain for stage
+            :param order
```

Comment:
Either you can some explanation, either don't add the parameter in docstring :D (here probably just tell that order and limit are given to ``search``).